tools/filelife.py

```diff
@@ -109,12 +151,21 @@
     if args.ebpf:
         exit()
 
+if BPF.kernel_struct_has_field(b'renamedata', b'old_mnt_userns') == 1:
```

Comment:
I checked the original patch (https://lore.kernel.org/all/20210121131959.646623-15-christian.brauner@ubuntu.com/) this looks good to me.